### ISSUE TITLE
BUG : `@antifeature`

### ISSUE THREAD

**Issue by Malix-Off** [lisonge/vite-plugin-monkey#61]
## Exemple

```ts
...
userscript: {
    ...
    antifeature: {
	    type: "tracking",
	    description: "test"
    },
    ...
},
...
```

build:
```js
// @antifeature  trackingtest
```

## Cause

#41

**Comment by lisonge** [comment]
Before the bug was not fixed, you can use

```txt
$extra: [['antifeature', 'ads         We show you ads']],
```

**Comment by lisonge** [comment]
fixed in [v3.0.2](https://github.com/lisonge/vite-plugin-monkey/blob/v3.0.2/packages/vite-plugin-monkey/CHANGELOG.md)
